pkg/operator/sync.go

```diff
@@ -560,6 +578,18 @@ func (optr *Operator) applyManifests(config *renderConfig, paths manifestPaths)
 		}
 	}
 
+	for _, path := range paths.configMaps {
```

Comment:
This could be useful for the on-cluster build work in the future! cc: @dkhater-redhat.